tools/namefixer.py

```diff
@@ -936,17 +936,20 @@
     "func_800E8EA0": "Actor_ContinueText",
     "func_800B90F4": "Actor_DeactivateLens",
 
-    "func_8010A000": "Map_GetDungeonOrBossAreaIndex",
-    "func_8010A074": "Map_IsInDungeonOrBossArea",
-    "func_8010A0F0": "Map_GetDungeonAreaIndex",
+    "func_8010A000": "Map_GetDungeonSceneIndex",
+    "func_8010A074": "Map_IsInDungeonOrBossScene",
+    "func_8010A0F0": "Map_GetDungeonSceneIndexForDungeon",
     "func_8010A164": "Map_IsInDungeonArea",
```

Comment:
```suggestion
    "func_8010A164": "Map_IsInDungeonScene",
```
namefixer doesn't like it if you have the same thing as both a key and a value, so should just update the value for the old entry.